news/views.py

```diff
@@ -712,6 +712,13 @@ def update_user_task(request, api_call_type, data=None, optin=False, sync=False)
                           rate=EMAIL_SUBSCRIBE_RATE_LIMIT, increment=True):
             raise Ratelimited()
 
+    if api_call_type == SET and token and data.get('newsletters'):
+        # only rate limit here so we don't rate limit errors.
+        if is_ratelimited(request, group='news.views.user',
```

Comment:
A fair point. I used `user` because that's the view from which these requests come. But using the type is probably more clear. Thanks.